source/MaterialXView/Main.cpp

```diff
@@ -102,6 +108,18 @@ int main(int argc, char* const argv[])
         {
             materialFilename = nextToken;
         }
+        else if (token == "--bakeWidth")
```

Comment:
As above, it looks like the existing pattern is to test for tokens in the same order that they're documented above, so "--bakeWidth" would be tested just after "--captureFilename".